experimenter/experimenter/targeting/constants.py

```diff
@@ -127,6 +127,8 @@ def __post_init__(self):
     {SPONSORED_SEARCH_SUGGESTIONS_DISABLED}
 )
 """
+# Most sponsored content is off by default in Brazil and Mexico
+ADS_DISABLED_BR_MX = TOPSITES_OR_SPONSORED_TOPSITES_DISABLED
```

Comment:
GIven that this stuff is inherently unstable because it changes over time, how about we name this (and future similar constants) more explicitly so that we don't inadvertently reuse it at some point without checking that things haven't changed:

ADS_DISABLED_BR_MX_2025_06

and update the callers?